packages/completer/style/base.css

```diff
@@ -280,6 +280,23 @@
   }
 }
 
+.jp-GhostText-errorIndicator::after {
+  animation: jp-GhostText-error 2s infinite;
+  animation-delay: 400ms;
+  content: ' ';
```

Comment:
Just the colour change feels a bit subtle. It is possibly fine if providers will also show the details some place else. Accessibility-wise we should not relay solely on colour. An alternative to only changing colour would be adding a non-empty content, for example ⚠ though i'm not sure if this would look nice.